web/src/HUD.tsx

```diff
@@ -64,8 +64,11 @@ class HUD extends Component<HudProps, HudState> {
     this.pathBuilder = new PathBuilder(window.location)
     this.controller = new AppController(this.pathBuilder, this)
     this.history = props.history
-    this.unlisten = this.history.listen((location, _) => {
-      let tags = { type: pathToTag(location.pathname) }
+    this.unlisten = this.history.listen((location: any, action: string) => {
```

Comment:
hmm can you explain what's going on here?

Reply:
ya, here's a good doc on how it works: https://github.com/ReactTraining/history/blob/v4.10.1/docs/GettingStarted.md#listening